pkg/cni/cilium/cilium.go

```diff
@@ -326,14 +326,21 @@ func GetAppInstallOverrideValues(cluster *kubermaticv1.Cluster, overwriteRegistr
 func ValidateValuesUpdate(newValues, oldValues map[string]any, fieldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 	// Validate immutability of specific top-level value subtrees, managed solely by KKP
+	exclusions := []exclusion{
+		{
+			fullPath:  "cni.chainingMode",
+			pathParts: strings.Split("cni.chainingMode", "."),
+		},
+		{
+			fullPath:  "ipam.operator.clusterPoolIPv4PodCIDR",
+			pathParts: strings.Split("ipam.operator.clusterPoolIPv4PodCIDR", "."),
+		},
+	}
```

Comment:
Aren't we still calling `strings.Split` every time here when constructing this `[]exclusion` slice, which is every time `ValidateValuesUpdate` is called? I think putting those paths in `[]string{"cni", "chainingMode"}` and `[]string{"ipam", "operator", "clusterPoolIPv4PodCIDR"}` directly is probably fine.

Reply:
Decided to put it directly, I guess it won't be often that we would actually use it